webportal/src/app/cluster-view/services/service-info.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+// This function will call kubernetes restful api to get node - podlist - label info, to support service view monitor page.
+function getServiceView(kubernetesUrl) {
```

Comment:
>getServiceView [](start = 9, length = 14)

could you modify and add [https://github.com/Microsoft/pai/blob/master/webportal/src/app/cluster-view/services/services.component.js](https://github.com/Microsoft/pai/blob/master/webportal/src/app/cluster-view/services/services.component.js) in this PR? I assume this function will be used there.

Reply:
ok i will move the logist to there

Reply:
Solved

Reply:
Fan, as hao's need, this will maintain as a standalone file.